README.md

```diff
@@ -200,6 +200,41 @@ token_url_params    map[audience:https://dev-example.us.auth0.com/api/v2/]
 type                Bearer
 ```
 
+
+#### Client credentials with Private Key JWT Authentication
+
+Store private key in a file (e.g., private_key.pem), then configure Okta server with private key
```

Comment:
I would like to first see here a one sentence description of what this is for, kind of parallel to the Device code flow above.  

If this functionality is limited only to Okta (I'm not sure if it is or not), then I think that the title of the section should also mention Okta.  If it is not limited to Okta, then I think the text should say "for example".

Oh, and now I see that this is pretty much an exact copy of the example down in the Okta section below.  Maybe this whole section shouldn't even be here.